assignments/objects.js

```diff
@@ -11,54 +11,176 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
+  id: 0,
+  name: 'Example',
+  email: 'examples@you.edu',
+  gender: 'F'
 }
 
 // Write your intern objects here:
+;`1,mmelloy0@psu.edu,Mitzi,F,
+2,kdiben1@tinypic.com,Kennan,M,
+3,kmummery2@wikimedia.org,Keven,M
+4,gmartinson3@illinois.edu,Gannie,M
+5,adaine5@samsung.com,Antonietta,F`
+  .split('\n')
+  .map(chunk => chunk.split(','))
```

Comment:
Nice use of .map, but the the assignment called for using for loops and not using .map, .filter,  .reduce. These were outside of the scope of the assignment.